modules/installation-configuration-parameters.adoc

```diff
@@ -1335,6 +1335,10 @@ in vSphere.
 |`platform.vsphere.ingressVIP`
 |The virtual IP (VIP) address that you configured for cluster ingress.
 |An IP address, for example `128.0.0.1`.
+
+|`platform.vsphere.diskType`
+|_Optional_. The disk provisioning method. This value defaults to the vSphere default storage policy if not set.
```

Comment:
Interesting that _optional_ is italicized in this table but not in Table. I've generally seen it without any formatting. Calling it out in case you feel like also changing it on line 1320 but that's your call.